source/getting-started/onboarding.markdown

```diff
@@ -3,56 +3,53 @@ title: "Onboarding Home Assistant"
-   - It is also used for location-based information and automations: for example showing the weather-forecast, opening the shades at sunrise, or starting the vacuum when you leave the home.
-   - If you'd rather not send your location, you can choose a location far away from where you live.
-   - You can always change this information later in the settings.
+**Step 3:** Enter the location of your home.
 
-    ![Define your location.](/images/getting-started/onboarding_location.png)
+The location is used to configure the time zone, unit system, and currency. It is also used for automations that would show the weather, open the shades at sunrise, or start the vacuum when you leave the home.
 
-5. Select which information you are willing to share.
-    - Sharing is disabled by default. However, we would like to encourage you to share some of this data.
-    - This information helps us to find out which platforms we need to support and where to focus our efforts.
-    - The data is anonymized and aggregated. To see the charts we generate out of this data, take a look at our [analytics page](https://analytics.home-assistant.io/).
-   ![Share anonymized data](/images/getting-started/onboarding_share_anonymized_info.png)
+You can always change this information later in settings.
+![Define your location.](/images/getting-started/onboarding_location.png)
 
-6. Once you are done, select **Next**.
-    - Home Assistant will then show any {% term devices %} it has discovered on your network.
-    - Don't be alarmed if you see fewer items than shown below; you can always manually add devices later.
+**Step 4:** Improving Home Assistant.
 
-    ![Discovery of devices on your network.](/images/getting-started/onboarding_devices.png)
+Home Assistant can use anonymous information about your system to improve the product. This is disabled by default. However, we would like to encourage you to share some of this data as it helps us to find out which platforms we need to support and where to focus our efforts.
 
-7. Finally, select **Finish**.
-   - Now you're brought to the Home Assistant web interface. If some of your devices were discovered and set up automatically, this default dashboard may already show some of your devices.
+The data is anonymized and aggregated. To see the charts we generate out of this data, take a look at our [analytics page](https://analytics.home-assistant.io/)
+
+Once you are done, select Next.
+![Share anonymized data](/images/getting-started/onboarding_share_anonymized_info.png)
+
+**Step 5: Show found devices.**
+
+Home Assistant will then try to find any devices it has discovered on your network. What shows up depends on the devices in your home. Don’t worry if you don’t see anything but your name at the top. You can add more devices later.
+
+Select Finish.
+![Discovery of devices on your network.](/images/getting-started/onboarding_devices.png)
+
+You’ve now got your basic Home Assistant server up and running.
```

Comment:
Same as before: We should avoid the term "server".

Reply:
Happy to make the change, just don't know what the preferred term should be.

Reply:
It can be dropped with loss of context imho

Reply:
To be clear, you're saying "You've now got your basic Home Assistant up and running" is preferred? That sounds a bit odd to me but if this is the accepted, and used, framing, I'll change it. It might make more sense to simplify it futher: "You now got Home Assistant up and running". Would that also work?